.lintr

```diff
@@ -1,5 +1,6 @@
 linters: linters_with_defaults(
   line_length_linter = line_length_linter(120),
   cyclocomp_linter = NULL,
-  object_usage_linter = NULL
+  object_usage_linter = NULL,
+  indentation_linter = NULL
```

Comment:
@kartikeyakirar is this needed? any way we can revert this indentation NULL?

Reply:
without this super linter checks are failing https://github.com/insightsengineering/teal.modules.general/actions/runs/6419052734/job/17428092698 thats I kept this.

Reply:
@kartikeyakirar I rather have you update the code, so that we follow new indentation rules, instead of omitting the check.
Please remove this indentation NULL in .lintr and please edit the code where we need to remove or append lines. Sorry

Reply:
I had previously attempted this, updating the code in accordance with linter warnings but github auto-action restyle the file.

Reply:
ok, let's leave it with indentation_linter = NULL